funnel/models/event.py

```diff
@@ -125,6 +134,24 @@ class Participant(BaseMixin, db.Model):
     __table_args__ = (db.UniqueConstraint('proposal_space_id', 'email'),)
 
     @classmethod
+    def get(cls, space, email, participant_attrs={}, create=False):
+        participant = cls.query.filter_by(proposal_space=space, email=email).first()
```

Comment:
The combination of (space, email) is enforced to be unique. If there are multiple entries in Explara with the same email for a given space, then it's taken to mean that, the particular participant has multiple tickets.